.gitattributes

```diff
@@ -1 +1,2 @@
 /testdata/** eol=lf
+* text=auto
```

Comment:
I suspect it might be this line that is breaking CI on Windows 10 and Windows Server 2012 R2.